src/smtp_util.erl

```diff
@@ -56,7 +56,7 @@ mxlookup(Domain) ->
 		[] ->
 			lists:map(fun(X) -> {10, inet_parse:ntoa(X)} end, inet_res:lookup(Domain, in, a));
 		Result ->
-			lists:sort(fun({Pref, _Name}, {Pref2, _Name2}) -> Pref =< Pref2 end, Result)
+			lists:sort(Result)
```

Comment:
There is no need for the sorting function, natural sorting will give the same effective result. A minor difference is that entries with the same preference will also be sorted by `Name`s, which is of no consequence.